src/pages/receipts/csvFileReceipt.js

```diff
@@ -144,6 +151,21 @@ const updateInTransitMapping = (shippedBoxes) => {
   return holdProcessedResult
 }
 
+
+/**
+ * Maps specimen id to material type based on last 4 digits
+ * @param {string} specimenId - Specimen id from each specimen bag
+ * @returns {string} Returns material type
+*/ 
+
+const materialTypeMapping = (specimenId) => {
+  const tubeId = specimenId.split(' ')[1]
+  const materialTypeObject = {'0001':'Serum', '0002':'Serum', '0011':'Serum', '0012':'Serum', '0021':'Serum', 
```

Comment:
NIT: Using object seems to better than the old way. But i do not know we should define it in `shared.js`?

Reply:
it can be moved to shared.js only if its used across multiple locations/functions